src/components/ApeTextField/ApeTextField.tsx

```diff
@@ -37,7 +39,7 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   inputRootError: {
-    border: `1px solid ${theme.colors.red}`,
+    border: `1px solid ${theme.colors.red}dd`,
```

Comment:
`dd` is that intentional?

Reply:
i know this is hacky, I don't like import transparentize to add a little transparency

Reply:
Ideally I'd like the theme to have `redFade80` or something -it'd be good to cleanup styles everywhere, and alx wants to change fonts even

Reply:
this was new to me too -- https://caniuse.com/?search=rrggbbaa

Reply:
Wow cool. @exrhizo I just didn't know about alpha RGBA CSS.